### PR TITLE
Add support for secrets as files

### PR DESCRIPTION
**Pull request by drgarcia1986**
🇯🇲

### CODE REVIEW

File: pkg/client/cmd/app.go [hunk 1]
```diff
@@ -533,20 +580,29 @@ func appEnvUnset(cmd *cobra.Command, args []string) {
...
 	Example: `  To add an new secret called "FOO":
 
   $ teresa app secret-set FOO=bar --app myapp
 
   You can also provide more than one env var at a time:
 
-  $ teresa app secret-set FOO=bar BAR=foo --app myapp`,
+  $ teresa app secret-set FOO=bar BAR=foo --app myapp
+
+  For file bases secrets use '-f' flag:
```
Comment (drgarcia1986) [inline pkg/client/cmd/app.go]: `s/bases/based/`

File: pkg/server/app/app.go [hunk 2]
```diff
@@ -70,6 +73,10 @@ type K8sOperations interface {
 	DeletePod(namespace, podName string) error
 	HasIngress(namespace, name string) (bool, error)
 	IngressEnabled() bool
+	CreateOrUpdateDeploySecretFile(namespace, deploy, fileName string) error
+	CreateOrUpdateCronJobSecretFile(namespace, cronjob, fileName string) error
```
Comment (drgarcia1986) [inline pkg/server/app/app.go]: `s/fileName/filename`

File: pkg/server/k8s/client.go [hunk 3]
```diff
@@ -1200,6 +1201,306 @@ func (c *Client) SetLoadBalancerSourceRanges(namespace, svcName string, sourceRa
 	return c.patchService(namespace, svcName, []byte(data))
 }
 
+func (c *Client) CreateOrUpdateDeploySecretFile(namespace, deploy, fileName string) error {
```
Comment (drgarcia1986) [inline pkg/server/k8s/client.go]: `s/fileName/filename`

File: pkg/server/k8s/client.go [hunk 4]
```diff
@@ -1200,6 +1201,306 @@ func (c *Client) SetLoadBalancerSourceRanges(namespace, svcName string, sourceRa
...
+	for _, vol := range vols {
+		if vol.Name == volName {
+			return vols
+		}
+	}
+	return append(
+		vols,
+		k8sv1.VolumeMount{
+			Name:      volName,
+			ReadOnly:  false,
```
Comment (drgarcia1986) [inline pkg/server/k8s/client.go]: `s/false/true` ???

File: pkg/server/app/app.go (hunk 2 above)
Comment (aguerra) [reply pkg/server/app/app.go]: both ok for me

**Comment by aguerra** [comment]
Great job. I would put it as: "Add support for secrets as files"

**Pull request merged**